networkx/algorithms/graphical.py

```diff
@@ -325,6 +356,17 @@ def is_digraphical(in_sequence, out_sequence):
     valid : bool
       True if in and out-sequences are digraphic False if not.
 
+    Examples
+    --------
+    >>> G = nx.DiGraph()
+    >>> G.add_edges_from([(1, 2), (1, 3), (2, 3), (3, 4), (4, 2), (5, 1), (5, 4)])
+    >>> in_deg = dict(G.in_degree())
+    >>> in_seq = list(in_deg.values())
+    >>> out_deg = dict(G.out_degree())
+    >>> out_seq = list(out_deg.values())
+    >>> nx.is_digraphical(in_seq, out_seq)
```

Comment:
```suggestion
    >>> G = nx.DiGraph([(1, 2), (1, 3), (2, 3), (3, 4), (4, 2), (5, 1), (5, 4)])
    >>> nx.is_digraphical([deg for _, deg in G.in_degree], [deg for _, deg in G.out_degree])
```
Using list comprehensions on getting the degrees makes this perhaps more simple?

Reply:
Okay sir I will simplify it.